### DISCUSSION TITLE
React Support

### DISCUSSION THREAD

**Discussion by imdkbj** [biati-digital/glightbox#363] [Ideas]
Request for support React and React Native with RTL.

**Comment by gingerchew** [comment]
Hey there @imdkbj 

You actually don't need anything special for React support, here's a [codepen](https://codepen.io/gingerchew/pen/dyePpoV) of the two working in tandem.

I can't speak for React Native, but if it's anything like React, see above. It's been said before though that GLightbox is meant for the web and for simplicity, so if you're trying to get *"That Native Feeling"* you might need to look for a different light-box library.
